apiclient/account.go

```diff
@@ -417,3 +419,61 @@ func (c *HTTPclient) RegisterSIKForVote(electionId types.HexBytes, proof *Census
 	}
 	return hash, nil
 }
+
+// ListAccounts return the account list information (address, balance and nonce)
+func (c *HTTPclient) ListAccounts(page int) ([]indexertypes.Account, error) {
+	resp, code, err := c.Request(HTTPGET, nil, "accounts", "page", strconv.Itoa(page))
+	if err != nil {
+		return nil, err
+	}
+	if code != apirest.HTTPstatusOK {
+		return nil, fmt.Errorf("%s: %d (%s)", errCodeNot200, code, resp)
+	}
+	accts := new(struct {
+		Accounts []indexertypes.Account `json:"accounts"`
+	})
+
+	if err := json.Unmarshal(resp, &accts); err != nil {
+		return nil, err
+	}
+
+	return accts.Accounts, nil
+}
+
+// AccountCount returns the total count of exiting accounts
+func (c *HTTPclient) AccountCount() (uint64, error) {
```

Comment:
i'd name this CountAccounts to be consistent with CountTokenTransfers  and ListAccounts